### DISCUSSION TITLE
Error while opening serialport 'Error: Error: Resource busy, cannot open /dev/ttyACM0'

### DISCUSSION THREAD

**Discussion by miltonleopoldo** [Koenkk/zigbee2mqtt#8326] [General]
Hi Guys,

I try to Home Assistant with Zigbee2mqtt on Raspery PI 4 B & ConBee II  and i'm stuck i can't discover what was the cause of the error:
Zigbee2MQTT:error 2021-08-10 21:46:11: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2021-08-10 21:46:11: Failed to start zigbee
Zigbee2MQTT:error 2021-08-10 21:46:11: Check https://www.zigbee2mqtt.io/information/FAQ.html#help-zigbee2mqtt-fails-to-start for possible solutions
Zigbee2MQTT:error 2021-08-10 21:46:11: Exiting...
**Zigbee2MQTT:error 2021-08-10 21:46:11: Error: Error while opening serialport 'Error: Error: Resource busy, cannot open /dev/ttyACM0'**
    at SerialPort.<anonymous> (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/znp/znp.ts:146:28)
    at SerialPort._error (/app/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:198:14)
    at /app/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:242:12
[21:46:12] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.21.0 start
> node index.js

Any tip to solve this?

**Comment by Koenkk** [comment]
I think it has the same cause as https://www.zigbee2mqtt.io/information/FAQ.html#error-resource-temporarily-unavailable-cannot-lock-port
